app/controllers/home_controller.rb

```diff
@@ -1,6 +1,5 @@
 class HomeController < ApplicationController
   before_action :authenticate_user!, only: [:index]
-  before_action :load_dwp_data, only: [:index], if: 'user_signed_in? && current_user.manager?'
   before_action :load_graph_data, only: [:index], if: 'user_signed_in? && current_user.admin?'
```

Comment:
Isn't it cleaner to call `load_graph_data` directly in the `index` action, rather then using filters? Especially if the filter has a complex condition and is used only in a single action.